### PR TITLE
Add x_dataset sample

### LINKED ISSUE #4: [DATASET] - X dataset request
**Name**
X Dataset

**Description**
This is a x dataset

**Motivation**
This dataset is awesome for image processing.

**Dataset Link**
awesome(dot)com/awesome

**Paper Link (if applicable)**
arxie/awesome

### PR DESCRIPTION
**Pull request by m3hrdadfi**
This dataset is related to this issue #4!

### CODE REVIEW

File: datasets/images/x_dataset/info.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+licenses:
+- mit
+size:
+- 10K<n<100K
+tasks:
+- image-captioning
+---
+
+# Name
+A short name for the dataset
```
Comment (baarsaam) [inline datasets/images/x_dataset/info.md:13]: incorrect name

**Review by baarsaam — CHANGES_REQUESTED** [review]
Change the file

File: datasets/images/x_dataset/info.md (hunk 1 above)
Comment (m3hrdadfi) [reply datasets/images/x_dataset/info.md:13]: Ok!

**Review by baarsaam — APPROVED** [review]
Thanks

**Pull request merged**